src/types.d.ts

```diff
@@ -529,3 +529,16 @@ export type KVBufferStore = {
   del(key: string): Promise<void>;
   has(key: string): Promise<boolean>;
 };
+
+/** Accepted in queue data item route fields as normalized b64 */
+export interface QueueDataItemHeaders {
```

Comment:
This type could be moved into `src/routes/ar-io.ts` since it's not part of a public interface.